.ci/build-app-image.sh

```diff
@@ -17,7 +17,11 @@ if [[ ! -d $SAMPLES_REPO/samples/$VERSION/$TEST_APP ]]; then
     exit 1
 fi
 cd  $SAMPLES_REPO/samples/$VERSION/$TEST_APP
-tee Dockerfile << EOF
+
+if [[ -f "Dockerfile" ]]; then
```

Comment:
I recall a previous discussion where you mentioned about this script enabling dnx trace...currently the samples in home repo do not enable it (and they should not ideally)...is this a concern?